modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/security/DoubleSignDetect.scala

```diff
@@ -0,0 +1,72 @@
+package org.tessellation.node.shared.domain.security
```

Comment:
I'd upvote my previous comment re: package name. IMO it should be snapshot or consensus-related, not in the general security

Reply:
Moved `DoubleSignDetect` to the `org.tessellation.node.shared.domain.snapshot` package.